pageserver/src/tenant.rs

```diff
@@ -3074,16 +3074,16 @@ mod tests {
 
     #[tokio::test]
     async fn test_prohibit_branch_creation_on_pre_initdb_lsn() -> anyhow::Result<()> {
-        let (tenant, ctx) =
```

Comment:
`ctx` should still be needed for line 3082?

Reply:
Yeah, I fumbled that while fixing conflicts on rebase. Fixed.